src/assets/i18n/en_US.json

```diff
@@ -1038,6 +1038,13 @@
   "product.short_description.label": "Short Description",
   "product.specific_attributes.label": "Specific Attributes",
   "product.variations.text": "Variations",
+  "product.warranty.code.text": "Warranty Code:",
+  "product.warranty.detail.text": "Details",
+  "product.warranty.expire_date.text": "Expires:",
+  "product.warranty.heading.text": "Protect your purchase:",
```

Comment:
Sounds like it was insecure to buy anything in our online shop. Would suggest rephrasing it to "Protect your purchased product" or some other headline that makes clear that this is about a warranty for products.

Reply:
Suggestion: "Product warranty options:",